### PR TITLE
Protocol Phantom Create: new protocol to create volume phantoms

### PR DESCRIPTION
**Pull request by EstrellaFG**
New protocol to create a volume phantom using xmipp_phantom_create program. Usefull protocol for testing (used in scipion tests of other protocols that are not already in devel).

### CODE REVIEW

**Comment by EstrellaFG** [comment]
code smell is outside my domain :)
Screenshot from 2021-11-25 13-54-25

**Comment by DavidHerreros** [comment]
> code smell is outside my domain :) @DavidHerreros ? Screenshot from 2021-11-25 13-54-25

Yes, this is an intended code smell. Most probably we will add that program in a future release when we have tested and updated it according to the new Zernike program changes. 

All the code related to that program is commented out so it is not offered in Scipion.

File: xmipp3/protocols/protocol_phantom_create.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    _label = 'phantom volume'
+
+    # --------------------------- DEFINE param functions ------------------------
+    def _defineParams(self, form):
+        form.addSection(label='Input')
+        form.addParam('desc', TextParam, label='Create phantom',
+                      default='40 40 40 0\ncyl + 1 0 0 0 15 15 2 0 0 0\nsph + 1 0 0 5 2\ncyl + 1 0 0 -5 2 2 10 0 90 0\n'
+                              'sph + 1 0 -5 5 2',
+                      help="create a phantom description: x y z backgroundValue geometry(cyl, sph...) +(superimpose) "
+                           "desnsityValue origin radius height rot tilt psi")
```
Comment (DStrelak) [inline xmipp3/protocols/protocol_phantom_create.py:45]: density
add documentation to the accepted params and format (probably doc of the program?)

**Review by DStrelak — CHANGES_REQUESTED** [review]

File: xmipp3/protocols/protocol_phantom_create.py (hunk 1 above)
Comment (EstrellaFG) [reply xmipp3/protocols/protocol_phantom_create.py:45]: There is not more information about phantom description in the original xmipp_phantom_create program. There are a couple of links (http://xmipp.cnb.csic.es/twiki/bin/view/Xmipp/FileFormats and http://newxmipp.svn.sourceforge.net/viewvc/newxmipp/trunk/testXmipp/input/phantom_descr.descr) but they are broken...

Comment (DStrelak) [reply xmipp3/protocols/protocol_phantom_create.py:45]: Can I ask you to add this to its documentation?
https://web.archive.org/web/20180813105422/http://xmipp.cnb.csic.es/twiki/bin/view/Xmipp/FileFormats

Comment (EstrellaFG) [reply xmipp3/protocols/protocol_phantom_create.py]: Done in https://github.com/I2PC/xmipp/pull/476

**Review by DStrelak — APPROVED** [review]

**Pull request merged**